examples/multi_label/classifier_chain.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
```

Comment:
I switched to yeast. I like it. It's much faster (the example runs in a few seconds). And it makes for a more convincing example.